pipeline-steps/deploy-network.yaml

```diff
@@ -36,7 +36,9 @@ steps:
          backendAzureRmResourceGroupName: 'azure-control-stg-rg'
       ${{ if eq( parameters['environment'], 'ptlsbox') }}: 
          backendAzureRmResourceGroupName: 'azure-control-sbox-rg'
-      ${{ if notIn( parameters['environment'], 'aat', 'perftest', 'ptlsbox') }}: 
+      ${{ if eq( parameters['environment'], 'ptl') }}: 
+         backendAzureRmResourceGroupName: 'azure-control-prod-rg'
```

Comment:
Is the `azure-control-prod-rg` references correct?

Reply:
yes.